metricbeat/module/ceph/config.go

```diff
@@ -0,0 +1,15 @@
+package ceph
+
+type Config struct {
+	CEPH *CephConfig `config:"ceph"`
+}
+
+type CephConfig struct {
+        BinaryPath             string `config:"binary_path"	validate:"required"`
```

Comment:
I don't think all these are required if we have defaults for them? If not configured we can fall back to the defaults, right?